component/azstorage/block_blob.go

```diff
@@ -475,8 +476,13 @@ func (bb *BlockBlob) getAttrUsingList(name string) (attr *internal.ObjAttr, err
 	var marker *string = nil
 	blobsRead := 0
 
+	maxResultsForList := bb.Config.maxResultsForList
```

Comment:
We can set the config value once during configure time and then use it in getAttrUsingList instead of checking it in everycall.